webcurator-webapp/src/main/webapp/jsp/target-instance-results.jsp

```diff
@@ -156,23 +156,31 @@
 											<c:forEach items="${reasons}" var="o">
 												<option value="<c:out value="${o.oid}"/>"><c:out value="${o.name}"/></option>
 											</c:forEach>
-										</select>&nbsp;    			
+										</select>&nbsp;
 						    		</c:when>
 					    		</c:choose>
 					    	</authority:hasPrivilege>
 					    	&nbsp;
 				    		</c:when>
 
-				    		<c:when test="${instance.state eq 'Patching' && hr.state eq 3}"> <!-- Indexing -->
+				    		<c:when test="${hr.harvestNumber ne 1 && hr.state eq 3}"> <!-- Indexing -->
                             <authority:hasPrivilege privilege="<%=Privilege.ENDORSE_HARVEST%>" scope="<%=Privilege.SCOPE_OWNER%>">
                                 <a href="#" onclick="javascript: return clickAbort(<c:out value="${hr.oid}"/>);">Abort</a>
-                                &nbsp;|&nbsp;
-                                <a href="#" onclick="javascript: return clickReIndex(<c:out value="${hr.oid}"/>);">Restart Indexing</a>
+                                <c:choose>
+                                	<c:when test="${instance.state eq 'Harvested'}">
+                                        &nbsp;|&nbsp;
+                                        <a href="#" onclick="javascript: return clickReIndex(<c:out value="${hr.oid}"/>);">Restart Indexing</a>
+                                    </c:when>
+                                    <c:when test="${instance.state eq 'Patching' && wct:xHoursElapsed(6,hr.creationDate)}">
```

Comment:
Should this time-out value of 6 (hours) be configurable or at least be defined as a constant somewhere?